web/util.ts

```diff
@@ -116,3 +116,9 @@ export function createElement(options: any): any {
     }
     return element;
 }
+
+export function toHex(x: number) {
```

Comment:
Please use a more specific and descriptive function name, such as "octetToHex" or "byteToHex", to explain how this function is different from only calling `number.toString(16)`.